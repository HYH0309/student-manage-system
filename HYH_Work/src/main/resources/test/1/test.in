2
1 0 1
20
3 10 7
2 2 4
