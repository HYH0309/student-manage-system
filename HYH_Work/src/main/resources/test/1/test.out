djwcb
-1
djwnb
5
